### DISCUSSION TITLE
Voice Processing Android

### DISCUSSION THREAD

**Discussion by rockoutloud11** [BearWare/TeamTalk5#1904] [Q&A]
### Description

Hello, So I'm not sure what is up but I have tried this on several Android devices. Voice Processing seems to not work. I can turn it on and nothing changes. I know on the Iphone it works really well, also with Windows. I have also tried the same thing on Chrome OS. I've tried this on a Pixel 6 Pro, 7 Pro, And Pixelbook. Any help would be greatly appreciated. Thank you.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [X] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

I expected it to filter out other users so they wouldn't feed back.

### Actual behavior

It doesn't seem like anything is happening. Everything sounds the same and everyone can hear themselves just like before.

### Steps to reproduce problem

Connect to a server.
Join a channel.
Go to Settings, Sound System, Enable Voice Preprocessing. Then exit out of Settings.

**Comment by bear101** [comment]
May be related to #1083
